src/components/EventCalender/Monthly/EventCalender.tsx

```
@@ -10,7 +10,7 @@
  * @param {InterfaceEvent[]} props.eventData - Array of event data to display.
  * @param {() => void} props.refetchEvents - Function to refetch events.
  * @param {InterfaceIOrgList} [props.orgData] - Organization data for filtering events.
- * @param {string} [props.userRole] - Role of the current user (e.g., SUPERADMIN, ADMIN).
+ * @param {UserRole} [props.userRole] - Role of the current user (ADMINISTRATOR or REGULAR).
```

ESLint

[error] tsdoc-param-tag-with-invalid-name: The @param block should be followed by a valid parameter name: The identifier cannot non-word characters

(tsdoc/syntax)